### PR TITLE
Add FSSAllBodiesFound

### PR DESCRIPTION
**Pull request by robbyxp1**
Second take on this PR, from develop, without the event field.

### CODE REVIEW

**Comment by robbyxp1** [comment]
#132 ref

File: schemas/fssallbodiesfound-v1.0.json [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+                    "format"        : "date-time",
+                    "description"   : "Timestamp upon receipt at the gateway. If present, this property will be overwritten by the gateway; submitters are not intended to populate this property."
+                }
+            }
+        },
+        "message": {
+            "type"                  : "object",
+            "description"           : "Contains all properties from the listed events in the client's journal, minus the Localised strings and the properties marked below as 'disallowed'",
+            "additionalProperties"  : false,
+            "required"              : [ "timestamp", "SystemName", "StarPos", "SystemAddress", "Count" ],
```
Comment (Athanasius) [inline schemas/fssallbodiesfound-v1.0.json:36]: `event` is no longer optional, so should be listed here again.

File: src/eddn/conf/Settings.py [hunk 2]
```diff
@@ -94,6 +97,8 @@ class _Settings(object):
         "http://schemas.elite-markets.net/eddn/blackmarket/1/test",
         "http://schemas.elite-markets.net/eddn/journal/1",
         "http://schemas.elite-markets.net/eddn/journal/1/test",
+        "http://schemas.elite-markets.net/eddn/fssallbodiesfound/1",
```
Comment (Athanasius) [inline src/eddn/conf/Settings.py:100]: No need to list any new schemas here.  It's purely so as to reject anyone still (how?) trying to send using the original-hostname schema URLs.

**Comment by Athanasius** [comment]
After me not noticing that this branch is `FSSAllBodiesFoundFromDevelop` rather than `FSSAllBodiesFound` ... `beta.eddn.edcd.io` is now up and running with this PR's code as of b7428d10c43a58cd13c8d72ce84b0f939def8362 .

My test message:

```json
{
  "$schemaRef": "https://eddn.edcd.io/schemas/fssallbodiesfound/1",
  "message": {
    "timestamp":"2022-01-26T16:21:00Z",
    "event":"FSSAllBodiesFound",
    "SystemName":"Zeta Doradus",
    "StarPos": [
      30.40625,-22.65625,-2.18750
    ],
    "SystemAddress":44853889387,
    "Count":1
  },
  "header": {
    "uploaderID": "from Athanasius Testing",
    "softwareName": "Athanasius Testing script",
    "softwareVersion": "v0.0.1"
  }
}
```
was accepted.

**Pull request merged**